frontend/components/FeatureCard/FeatureCard.tsx

```diff
@@ -0,0 +1,52 @@
+import "./featureCard.module.css";
+
+import { Box as ChakraBox } from "@chakra-ui/react";
+import { Center, ChakraProvider, Icon, Text } from "@chakra-ui/react";
```

Comment:
ChakraProvider is not needed as an import, we have everything handled in `preview.tsx` and `_app.tsx`